test/routes/appendix/appendix.test.js

```diff
@@ -115,6 +115,106 @@ describe('/appendix', () => {
     });
   });
 
+  describe('POST', () => {
+    test('/ - 200 Success with project/template combo', async () => {
+      const template1 = await db.models.template.create({name: uuidv4(),
+        organization: 'Test Org',
+        sections: ['summary', 'microbial']});
+
+      const res = await request
+        .post(`/api/appendix?templateId=${template1.ident}&projectId=${project.ident}`)
+        .auth(username, password)
+        .type('json')
+        .expect(HTTP_STATUS.CREATED);
+
+      expect(res.body).not.toBeNull();
+      await template1.destroy({force: true});
+      await db.models.templateAppendix.destroy({where: {ident: res.body.ident}, force: true});
+    });
+
+    // TODO failing
+    test('/ - 200 Success with null-project/template combo', async () => {
+      const template2 = await db.models.template.create({name: uuidv4(),
+        organization: 'Test Org',
+        sections: ['summary', 'microbial']});
+      const res = await request
+        .post(`/api/appendix?templateId=${template2.ident}`)
+        .auth(username, password)
+        .type('json')
+        .expect(HTTP_STATUS.CREATED);
+
+      expect(res.body).not.toBeNull();
+      await template2.destroy({force: true});
+      await db.models.templateAppendix.destroy({where: {ident: res.body.ident}, force: true});
+    });
+
+    // TODO failing
+    test('/ - 200 Success - by manager', async () => {
+      const template1 = await db.models.template.create({name: uuidv4(),
+        organization: 'Test Org1',
+        sections: ['microbial']});
+
+      const res = await request
+        .post(`/api/appendix?templateId=${template1.ident}&projectId=${project.ident}`)
+        .auth(managerUsername, password)
+        .type('json')
+        .send({...UPDATE_DATA2})
+        .expect(HTTP_STATUS.CREATED);
+
+      expect(res.body).not.toBeNull();
+      checkTemplateAppendix(res.body);
+      await template1.destroy({force: true});
+      await db.models.templateAppendix.destroy({where: {ident: res.body.ident}, force: true});
+    });
+
+    test('/ - 400 bad request due to non-unique project/template combo', async () => {
+      const template2 = await db.models.template.create({name: uuidv4(),
+        organization: 'Test Org',
+        sections: ['microbial']});
+
+      const res = await request
+        .post(`/api/appendix?templateId=${template2.ident}&projectId=${project.ident}`)
+        .auth(username, password)
+        .type('json')
+        .expect(HTTP_STATUS.CREATED);
+
+      expect(res.body).not.toBeNull();
+
+      await request
+        .post(`/api/appendix?templateId=${template2.ident}&projectId=${project.ident}`)
+        .auth(username, password)
+        .type('json')
+        .expect(HTTP_STATUS.CONFLICT);
+
+      await template2.destroy({force: true});
+      await db.models.templateAppendix.destroy({where: {ident: res.body.ident}, force: true});
+    });
+
+    // TODO failing
```

Comment:
Leftover TODO for obviously passing test

Reply:
ah ty